README.md

```diff
@@ -21,6 +21,8 @@ This is a capstone project, a website for a global concert.
 
 ### Live Link 
 
+https://bludidi.github.io/capstone-project/
+
```

Comment:
## OPTIONAL: In the acknowledgment section
- Please connect a link to  #MIT# license: Make it look like this 👉 This project is [MIT](https://github.com/microverseinc/readme-template/blob/master/LICENSE) licensed.